src/routes/role-router.js

```diff
@@ -2,37 +2,107 @@
 
 const express = require('express');
 const router = express.Router();
+const auth = require('../middleware/auth');
+const error403 = require('../middleware/403');
+const error401 = require('../middleware/401');
+const authorize = require('../middleware/authorize');
 
-// TODO: Swagger Comments
-// Visible by all clients
+
+
+/**
+ * Public is available to any user
+ * @route GET public
+ * @returns {object} 200 - Notifies the user they are allowed to access this route
+ */
 router.get('/public', (req, res, next) => {
   res.status(200).json({ valid: true });
 });
 
+router.use(auth);
+router.use(error401);
```

Comment:
smart to call these here instead of putting them in each of the remaining routes.